app/services/follow_remote_account_service.rb

```diff
@@ -16,7 +16,7 @@ def call(uri)
     return Account.find_local(username) if TagManager.instance.local_domain?(domain)
 
     account = Account.find_remote(username, domain)
-    return account unless account.nil?
+    return account unless account&.last_webfingered_at.nil? || account.last_webfingered_at + 86_400 < Time.now.utc
```

Comment:
Won't that do the opposite? If last_webfingered_at is nil, it will be compared in the second condition, and if it's anything truthy, it will trigger a re-fetch, even if we haven't timed out yet?